docs/products/kafka/concepts/acl.md

```diff
@@ -1,116 +1,216 @@
+### ACL Structure
 
--   `?*` matches a single character, and zero or more other
-    characters (equivalent to regular expression `(.+)`)
+An Aiven ACL entry consists of the following elements:
 
-Aiven for Apache Kafka® evaluates each topic access against the ACL
-entries. If it finds a matching ACL entry, access is granted. If no
-entry matches, access is denied. The order of the ACL entries is
-irrelevant.
+- **Username**: The Aiven for Apache Kafka service username or a wildcard pattern.
+- **Permission**: One of `read`, `write`, `readwrite`, or `admin`.
+- **Associated topics**: Specific Kafka topics or wildcard patterns.
 
-Examples:
+Aiven checks ACL entries and grants access if there's a match. If no entry matches,
+access is denied. The order of ACL entries does not
+influence access evaluation.
 
--   username: `abc`, permission: `read`, topic: `xyz`. User `abc` has
-    read access to topic `xyz`.
--   username: `analyst*`, permission: `read`, topic: `xyz`. All Aiven
-    users with username starting `analyst` have read access to topic
-    `xyz`.
--   username: `developer*`, permission: `read`, topic: `test*`. All
-    Aiven users with username starting `developer` have read access to
-    topics starting with `test`.
+### Examples
 
-:::warning
-By default, Aiven adds an `avnadmin` service user to every new service
-and adds `admin` permission for all topics to that user. When you create
-your own ACLs to restrict access, you probably want to remove this ACL
-entry.
-:::
+- User-specific access:
+
+  ```plaintext
+  username: abc
+  permission: read
+  topic: xyz
+  ```
+
+  Grants user `abc` read access to the topic `xyz`.
+
+- Wildcard-based access:
+
+  ```plaintext
+  username: analyst*
+  permission: read
+  topic: xyz
+  ```
+
+  Grants all users with usernames starting with `analyst` read access to the topic `xyz`.
+
+- Wildcard in topics:
+
+  ```plaintext
+  username: developer*
+  permission: read
+  topic: test*
+  ```
+
+  Grants all users with usernames starting with `developer` read access to topics
+  starting with `test`.
+
+## Kafka-native ACL capabilities
+
+Kafka-native ACLs provide granular, resource-level permissions for precise access
+control. They enable the management of Apache Kafka topics and additional resources
+like clusters, consumer groups, and transactional IDs. It is ideal for
+advanced scenarios requiring fine-tuned access management.
+
+- **Fine-grained permissions**: Allow both `ALLOW` and `DENY` rules to provide
+  precise control over access.
+- **Expanded resource-level control**: Manage access to non-topic resources,
+  such as consumer groups, clusters, and transactional IDs.
+- **Pattern-based matching**: Specify how resource names are matched using `LITERAL`
+  for exact matches or `PREFIXED` for prefixes.
+
+### ACL structure
+
+A Kafka-native ACL entry consists of the following elements:
+
+- **Principal**: Specifies the user or service account, such as `User:Alice`.
```

Comment:
One can use wildcards here, e.g. `User:alice*` for all usernames starting with `alice`. Also only `User:` is supported for now. Not sure how to explain those...

Reply:
Thanks, I updated the text here.